tests/test_pack.py

```diff
@@ -48,20 +46,20 @@ def test_random_boxes(self):
             str((Part() + packed).bounding_box()),
         )
 
-    def test_random_slots(self):
-        """Test pack for 2D objects."""
-        random.seed(123456)
-        # 50 is an arbitrary number that is large enough to exercise
-        # different aspects of the packer while still completing quickly.
-        inputs = [
-            SlotOverall(random.randint(1, 20), random.randint(1, 20)) for _ in range(50)
-        ]
-        # Not raising in this call shows successfull non-overlap.
-        packed = pack(inputs, 1)
-        self.assertEqual(
-            "bbox: 0.0 <= x <= 124.0, 0.0 <= y <= 105.0, 0.0 <= z <= 0.0",
-            str((Sketch() + packed).bounding_box()),
-        )
+
+def test_random_slots():
+    """Test pack for 2D objects."""
+    random.seed(123456)
+    # 50 is an arbitrary number that is large enough to exercise
+    # different aspects of the packer while still completing quickly.
+    widths = [random.randint(2, 20) for _ in range(50)]
+    heights = [random.randint(1, width - 1) for width in widths]
+    inputs = [SlotOverall(width, height) for width, height in zip(widths, heights)]
+    # Not raising in this call shows successfull non-overlap.
+    packed = pack(inputs, 1)
+    bb = (Sketch() + packed).bounding_box()
+    assert bb.min == Vector(0, 0, 0)
```

Comment:
These aren't valid unittest assertions. However, testing the min/max of the bounding box is better than a string compare.

Reply:
At least in ci y'all are using pytest to run your tests. Pytest will do the [right thing](https://docs.pytest.org/en/stable/how-to/assert.html#assert) with these assert statements.

Reply:
However, the test file itself only imports unittest and if run stand alone does:
```py
if __name__ == "__main__":
    unittest.main()
```
Let's make this file consistent at least with a unittest assertion.

Reply:
I started off exclusively using unittest but over time others have used pytest. When one receives generous donations in the form of tested code, best not to get into wars about what is "better".